services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/ConverterUtils.java

```diff
@@ -53,14 +53,19 @@ public static void validateFloat(Float input) {
     }
 
     public static String padLeft(int paddingAmount, int valueToPad) {
-        String value = Integer.toString(valueToPad);
-        int padding = paddingAmount - value.length();
-        StringBuilder result = new StringBuilder(paddingAmount);
-        for (int i = 0; i < padding; i++) {
-            result.append('0');
+        final String result;
+        final String value = Integer.toString(valueToPad);
```

Comment:
Our checkstyle rules do not allow final on local variables. Can you update?

Reply:
Done! Thanks for looking at this PR.